src/main/java/io/codelex/arrays/practice/Exercise6.java

```diff
@@ -0,0 +1,24 @@
+package io.codelex.arrays.practice;
+
+import java.util.Arrays;
+
+public class Exercise6 {
+    public static void main(String[] args) {
+
+        int[] randomNumberArray = new int[10];
+        int[] randomNumberArray2 = new int[randomNumberArray.length];
+
+        for (int i = 0; i < randomNumberArray.length; i++) {
+            randomNumberArray[i] = (int) (Math.random() * 100) + 1;
+        }
+
+        for (int i = 0; i < randomNumberArray.length; i++) {
+            randomNumberArray2[i] = randomNumberArray[i];
```

Comment:
ir arī `clone over arraycoppy` built in metodes, atvieglos dzīvi

Reply:
Tagad zinu - paldies. 😄
Izlabots.